tox.ini

```diff
@@ -17,7 +17,7 @@ deps =
     pytest-xdist
     coveralls
 commands =
-    check-manifest --ignore tox.ini,tests*,docs*
+    check-manifest --ignore tox.ini,tests*,docs*,*/**/*.pyc,*/**/__pycache__
```

Comment:
I never needed these, but *shrug*

Reply:
I needed them after the initial run of `tox`.